pkg/filesystem/path/windows_format.go

```diff
@@ -4,6 +4,8 @@ import (
 	"strings"
 
 	"github.com/buildbarn/bb-storage/pkg/util"
+	"google.golang.org/grpc/codes"
```

Comment:
Very tiny nit: could you add an empty line above this line? Right now we try to keep imports grouped by 2nd level domain. "Why?" is a good question, but staying consistent makes sense.